systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java

```diff
@@ -877,6 +879,93 @@ void testTlsHostnameVerificationWithMirrorMaker() {
+        final String kafkaUserWrite = "kafka-user-write";
+        final String kafkaUserRead = "kafka-user-read";
+        final String topicName = "my-topic-name-1";
+        final int numberOfMessages = 500;
+
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME,  3, 1)
+            .editMetadata()
+                .addToLabels("type", "kafka-ephemeral")
+            .endMetadata()
+            .editSpec()
+                .editKafka()
+                .withNewKafkaAuthorizationSimple()
+                .endKafkaAuthorizationSimple()
+                .editListeners()
+                    .withNewKafkaListenerExternalNodePort()
+                        .withNewKafkaListenerAuthenticationTlsAuth()
+                        .endKafkaListenerAuthenticationTlsAuth()
+                    .endKafkaListenerExternalNodePort()
+                .endListeners()
+                .endKafka()
+            .endSpec()
+            .done();
+
+        testMethodResources().topic(CLUSTER_NAME, topicName).done();
+
+        testMethodResources().tlsUser(CLUSTER_NAME, kafkaUserWrite)
+                .editSpec()
+                    .withNewKafkaUserAuthorizationSimple()
+                        .addNewAcl()
+                            .withNewAclRuleTopicResource()
+                                .withName(topicName)
+                            .endAclRuleTopicResource()
+                            .withOperation(AclOperation.WRITE)
+                        .endAcl()
+                        .addNewAcl()
+                            .withNewAclRuleTopicResource()
+                                .withName(topicName)
+                            .endAclRuleTopicResource()
+                            .withOperation(AclOperation.DESCRIBE)  // describe is for that user can find out metadata
+                        .endAcl()
+                    .endKafkaUserAuthorizationSimple()
+                .endSpec()
+                .done();
+
+        StUtils.waitForSecretReady(kafkaUserWrite);
+
+        LOGGER.info("Checking kafka user:{} that is able to send messages to topic:{}", kafkaUserWrite, topicName);
+
+        sendMessagesExternalTls(NAMESPACE, topicName, numberOfMessages, kafkaUserWrite);
+
+        assertThrows(ExecutionException.class, () -> receiveMessagesExternalTls(NAMESPACE, topicName, numberOfMessages, kafkaUserWrite));
+
+        testMethodResources().tlsUser(CLUSTER_NAME, kafkaUserRead)
+                .editSpec()
+                    .withNewKafkaUserAuthorizationSimple()
+                        .addNewAcl()
+                            .withNewAclRuleTopicResource()
+                                .withName(topicName)
+                            .endAclRuleTopicResource()
+                            .withOperation(AclOperation.READ)
+                        .endAcl()
+                        .addNewAcl()
+                            .withNewAclRuleGroupResource()
+                                .withName("*")
```

Comment:
while maybe I am fine with using just "*" as consumer group name in this test, I would enable to use a specific group name. While it's simpler in the Acl declaration here, I see that down the methods chain, the `createConsumerProperties` just generate a random id for a consumer group that of course you don't know here to set into the Acls. I would enable the test to inject the consumer group to use from the top. I think we could need that for more tests on KafkaUser acls and so on.